File: src/lib/scala/Slate.Core/src/main/scala/slate/core/app/AppOptions.scala
/**
<slate_header>
  url: www.slatekit.com
  git: www.github.com/code-helix/slatekit
  org: www.codehelix.co
  author: Kishore Reddy
  copyright: 2016 CodeHelix Solutions Inc.
  license: refer to website and/or github
  about: A Scala utility library, tool-kit and server backend.
  mantra: Simplicity above all else
</slate_header>
  */
package slate.core.app

case class AppOptions (
                        printSummaryBeforeExec :Boolean = false,
                        printSummaryOnShutdown :Boolean = true ,
                        getConfNameFromEnv     :Boolean = true ,
                        getLogNameFromEnv      :Boolean = true
                      )
{
}

File: src/lib/kotlin/slatekit-app/src/main/kotlin/slatekit/app/AppOptions.kt
/**
 * <slate_header>
 * url: www.slatekit.com
 * git: www.github.com/code-helix/slatekit
 * org: www.codehelix.co
 * author: Kishore Reddy
 * copyright: 2016 CodeHelix Solutions Inc.
 * license: refer to website and/or github
 * about: A tool-kit, utility library and server-backend
 * mantra: Simplicity above all else
 * </slate_header>
 */

package slatekit.app

open class AppOptions(
    val showWelcome: Boolean = true,
    val showDisplay: Boolean = true,
    val showSummary: Boolean = true,
    val getConfNameFromEnv: Boolean = true,
    val getLogNameFromEnv: Boolean = true
)
